### PR TITLE
test(manager): introduce Manager installation test on Rocky8 distribution

### PR DESCRIPTION
**Pull request by mikliapko**
This PR introduces the Manager installation test on Rocky8 distro (https://github.com/scylladb/scylla-manager/issues/3854).

Contains:
- Configuration file for Rocky8 distro;
- jenkinsfile for Rocky installation test;
- some adjustments to monitoring node configuration methods to make installation on Rocky finctuonal.

Test triggers were adjusted as well:
https://github.com/scylladb/scylla-pkg/pull/4137

### Testing

- [x] https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/rocky8-installation-test/3/

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: configurations/manager/rocky8.yaml
Comment (mikliapko) [inline configurations/manager/rocky8.yaml:1]: I suppose we should use the approach with resolve:ssm parameter introduced recently.
I'm not sure I have permissions to create such parameter:
```
$ aws ssm put-parameter --name "/aws/service/rocky/release/8/latest/x86_64" --value "ami-011ef2017d41cb239" --type "String"

An error occurred (AccessDeniedException) when calling the PutParameter operation: No access to "/aws/" namespace: aws/service/rocky for the account: 797456418907
```

@fruch Could you please help with that?

**Comment by karol-kokoszka** [comment]
@mikliapko did you have a chance to execute it against latest manager-master  ?
Can we see the results ?

**Comment by mikliapko** [comment]
> @mikliapko did you have a chance to execute it against latest manager-master ? Can we see the results ?

Sure, we can. I added the job in manager-master Jenkins space (it lokks into my fork, will fix it after merge):
https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/view/Installation/job/rocky8-installation-test/

Also, just an hour ago I already did a run for this test with manager latest - it passed:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/rocky8-installation-test/3

**Review by karol-kokoszka — APPROVED** [review]

**Review by Michal-Leszczynski — APPROVED** [review]

File: configurations/manager/rocky8.yaml
Comment (fruch) [reply configurations/manager/rocky8.yaml:1]: The idea with those is to read the official image of a disto (if it has ones)

Maybe we can put values into SSM, but we never intended to do so.

In theory every thing in the marketplace should be searchable via its product id, i.e. as I'm doing for Ubuntu, but I haven't found a clear way of obtaining it from the marketplace (didn't have time to dig deeper)

File: sdcm/cluster.py
Comment (fruch) [inline sdcm/cluster.py:2234]: Can be replaced with self.install_package

File: sdcm/cluster.py
Comment (fruch) [inline sdcm/cluster.py:5398]: Can we not inline this if ? So it would be a bit more readable, and assign to the variable as needed ?

File: configurations/manager/rocky8.yaml
Comment (fruch) [inline configurations/manager/rocky8.yaml:3]: We switch to branch-4.7

File: configurations/manager/rocky8.yaml
Comment (mikliapko) [reply configurations/manager/rocky8.yaml:1]: So, I've tried to search for it as well - didn't find anything for Rocky describing public parameters (https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-finding-public-parameters.html).

Therefore, I'm keeping the ami_id in the configuration file.

File: sdcm/cluster.py
Comment (vponomaryov) [reply sdcm/cluster.py:2234]: +1

File: sdcm/cluster.py
Comment (vponomaryov) [inline sdcm/cluster.py:5405]: `pip`-specific actions are not required for non-rocky distr.
So, need to make it be conditioned to avoid unnecessary breakages if pip code gets broken for any reason.

File: configurations/manager/rocky8.yaml
Comment (vponomaryov) [reply configurations/manager/rocky8.yaml:1]: Better to specify region in comment too.

File: configurations/manager/rocky8.yaml
Comment (vponomaryov) [reply configurations/manager/rocky8.yaml:3]: +1

**Comment by mikliapko** [comment]
All the comments have been addressed, retesting here:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/rocky8-installation-test/4/

**Comment by mikliapko** [comment]
@scylladb/qa-maintainers Could you please take a look once again after all the comments have been addressed?

File: sdcm/cluster.py
Comment (fruch) [inline sdcm/cluster.py:5476]: nitpick: isn't those command also works on all of the rest rhel_like ?

**Review by fruch — APPROVED** [review]
LGTM

File: sdcm/cluster.py
Comment (mikliapko) [reply sdcm/cluster.py:5476]: I believe it should but I haven't checked it and decided to go with the approach above to minimize possible regression.

**Pull request merged**
